src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -494,13 +452,6 @@ void ResourceAllocationManagerImpl::OnPolicyEvent(
     ProcessApplicationPolicyUpdate();
     return;
   }
-
-  if (PolicyEvent::kApplicationsDisabled == event) {
-    ResetAllAllocations();
```

Comment:
@LuxoftAKutsan please add handling of PolicyEvent::kApplicationsDisabled event  to InteriorDataManagerImpl::OnPolicyEvent(plugins::PolicyEvent event)